### ISSUE TITLE
General questions

### ISSUE THREAD

**Issue by BenTheQ** [fokhruli/STGCN-rehab#1]
- How is the data preprocessing for the KIMORE dataset done? Would like to know so we can adapt it for other data if possible
- I noticed from data_preprocessing.py that it imports Train_X.csv and Train_Y.csv, possible to share the link to the processed-data.zip? 
- The GoogleDrive link on github is not working
- For X_train do you only use the joint positions?
- In the KIMORE dataset there are 5 exercises, does this mean that for each exercise there is a separate trained GCN model (total 5 model for 5 exercise)?
- May I know what the pretrained model (best_model.hdf5) provided in the github was trained on? (All the intances in KIMORE dataset for 1 exercise?)

**Comment by fokhruli** [comment]
Please, find reply of queries below:

1) The data processing step was done following the [KIMORE dataset](https://vrai.dii.univpm.it/content/kimore-dataset) paper and [github](https://github.com/petteriTeikari/KiMoRe_wrapper). Then we use ''data_processing.py'' file for pre-processing the joint data. For each joint, we have (x,y,z) position along the z axis, joint number along y axis and finally timestep along x axis. The ''data_processing.py'' file will output the aforementioned data structure.

2) We added preprocessed dataset link in github repo. Please, check it now.

3) Yes, for X_train we only use the joint position. However, you can also use joint orientation along with the joint position to make the model robust.

4) Since the exercises differ from each other in various form, we train the network for each of the exercises separately.

5) ''best_model.hdf5'' is the weight of our network for the best performing model among 1500 epochs. Yes, it was for KIMORE ex5. But you can find all exercise [here](https://drive.google.com/drive/folders/182buX7plr7Cs3Y74hfhfZXiTictukqTZ?usp=sharing).

We will update and organize (with rgb evaluation code) our github repository very soon.
